package.json

```diff
@@ -16,7 +16,9 @@
     "next": "13.2.3",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-uid": "^2.3.2",
```

Comment:
You probably don't need this package as it is for generating stable `key` values across server/client renders.

Reply:
You are right, i was confused and download the wrong and then forgot to erased it, actually i don't know how to erase it...

Reply:
Done.